### ISSUE TITLE
Compatibility with BERT models

### ISSUE THREAD

**Issue by shivam-chandhok** [facebookresearch/SentEval#78]
Is there a way to probe for BERT based models using this framework?

**Comment by laurinehu** [comment]
Yes, just edit **prepare** and **batcher** (see one of the examples in SentEval/examples/) so that they build your embedding based on your BERT based model

**Comment by Sandy4321** [comment]
there no such an example fro bert ?
where is it?

**Comment by Sandy4321** [comment]
laurinehu commented on Aug 12, 2020
what is 3 years ago
but last code change
in
https://github.com/facebookresearch/SentEval/tree/main/examples
is 5 years ago
